calibration/CTDBPC/CGINS-CTDBPC-07208__20190324.csv

```diff
@@ -0,0 +1,23 @@
+serial,name,value,notes
+16-7208,CC_a0,1.239424e-003,
+16-7208,CC_a1,2.764232e-004,
+16-7208,CC_a2,-1.218914e-006,
+16-7208,CC_a3,1.875789e-007,
+16-7208,CC_cpcor,-9.57E-08,
+16-7208,CC_ctcor,3.25E-06,
+16-7208,CC_g,-9.880803e-001,
+16-7208,CC_h,1.266653e-001,
+16-7208,CC_i,-1.647309e-004,
+16-7208,CC_j,2.868731e-005,
+16-7208,CC_pa0,6.073118e-003,
+16-7208,CC_pa1,4.834518e-004,
+16-7208,CC_pa2,-5.611719e-012,
+16-7208,CC_ptca0,5.251666e+005,
+16-7208,CC_ptca1,6.833663e+000,
+16-7208,CC_ptca2,-2.875111e-001,
+16-7208,CC_ptcb0,2.525700e+001,
+16-7208,CC_ptcb1,-8.000000e-004,
+16-7208,CC_ptcb2,0.000000e+000,
+16-7208,CC_ptempa0,-6.115465e+001,
+16-7208,CC_ptempa1,5.273134e+001,
+16-7208,CC_ptempa2,-5.294703e-001,
```

Comment:
Verified that values taken from incoming vendor .cal file used to create csv match those on QCT 3305-00102-00212-A.txt file.

Reply:
Confirmed.